plugin-docker/che-plugin-docker-machine/src/test/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProviderTest.java

```diff
@@ -1015,7 +1015,7 @@ public void shouldAddExtraHostOnNonDevInstanceCreationFromSnapshot() throws Exce
                                                             Collections.emptySet(),
                                                             devVolumes,
                                                             commonVolumes,
-                                                            new String[]{"dev.box.com:192.168.0.1", "codenvy.com.com:185"},
+                                                            "dev.box.com:192.168.0.1,codenvy.com.com:185",
```

Comment:
this is in purpose
, is delimiter for String split and I want to avoid extra space